src/app/body/task-details/task-details.component.html

```diff
@@ -308,4 +308,7 @@ <h4 id="value">{{task.Title}}</h4>
         <app-git [taskId]="Id" [orgDomain]="orgDomain" [task]="task" (addedPrLink)="addedPrLink($event)"></app-git>
     </ng-container>
 </ng-container>
-<app-loader *ngIf="!(gotTaskData)"></app-loader>
\ No newline at end of file
+<app-loader *ngIf="!(gotTaskData)"></app-loader>
+<ng-container *ngIf="showLoader">
+    <app-loader></app-loader>
+</ng-container>
```

Comment:
Rather than using a new app loader with different conditional statement, Use one variable to handle both. gotTaskData and showLoader